### PR TITLE
Added list of specification maintainers

### PR DESCRIPTION
**Pull request by cyenyxe**
List of maintainers in `MAINTAINERS.md`. Reference to said list in `README.md`, with a gentle reminder to use GitHub in the first place.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+### SAM/BAM
+
+* James Bonfield (@jkbonfield)
+* Yossi Farjoun (@yfarjoun)
+
+### CRAM
+
+* James Bonfield (@jkbonfield)
+* Vadim Zalunin (@vadimzalunin)
```
Comment (droazen) [inline MAINTAINERS.md]: I believe that @vadimzalunin will be departing from his role as CRAM maintainer in the near future. @vadimzalunin Do you have a replacement lined up?

**Comment by droazen** [comment]
One comment/question, otherwise looks good to me.

File: MAINTAINERS.md (hunk 1 above)
Comment (tk2) [inline MAINTAINERS.md]: Indeed he is, we have a transition plan at EBI via @AlexanderSenf and @AlexanderVi
For now, it is fine to list Vadim.

**Comment by tk2** [comment]
@cyenyxe can you add @jmarshall as co-maintainer of SAM/BAM also.

**Comment by jmarshall** [comment]
The text added to README should probably mention issues and/or PRs.

Currently this text also appears on [the website](http://samtools.github.io/hts-specs/), but it's not needed there.  Adjust the `offset` at the bottom of _index.md_ to hide it.

**Comment by jmarshall** [comment]
I've pushed the website formatting fix to add to your branch squashage.

I agree with James that it would be good to list the historical editors too.  In the commit message would suffice, or the madly-overcorrect approach would be a series of commits reflecting the maintainership changes :smile:

**Comment by cyenyxe** [comment]
Thanks @jmarshall, I have squashed your changes.

Please somebody merge this and add the list of historical editors to the merge commit message, because I only know about Heng :sweat_smile:

**Comment by jmarshall** [comment]
I have some notes from when I've tracked this down in the past.

**Comment by cyenyxe** [comment]
If there are no complaints about merging this without the list of historical maintainers, I will do so by the end of this week.

**Comment by jmarshall** [comment]
Added GA4GH history.

The original SAM1.pages lists the following as contributors:

> The major contributors to this specification are (in no particular order): Heng Li (Sanger Institute),
Bob Handsaker (Broad Institute), Jue Ruan (Beijing Genomics Institute), Richard Durbin (Sanger Institute), Gabor Marth (Boston College), Michael Stromberg (Boston College), Fiona Hyland (Applied Biosystems), Goncalo Abecasis (University of Michigan), Richa Agarwala (NCBI)

If we wanted to say something about G1K-era pre-GA4GH history, we could summarise that or just list Heng, and we'd want to recall the equivalent list of contributors or a main editor for VCF.  Or we could just leave this note on the PR :smile:.

**Pull request merged**